tools/trace_analyzer_test.cc

```diff
@@ -37,7 +37,7 @@ namespace rocksdb {
 namespace {
 static const int kMaxArgCount = 100;
 static const size_t kArgBufferSize = 100000;
-}
+}  // namespace
```

Comment:
did the linter add this? cool

Reply:
I don't remember -__-b
Thanks for your suggestion of `git add -p`